README.md

```diff
@@ -1,8 +1,73 @@
 # orb_slam_2_ros
-This package integrates orb_slam_2 into ROS in what I believe to be a more user friendly way than what is offered by the underlying library found here:
 
-https://github.com/raulmur/ORB_SLAM2
+## Overview
 
-Depends on my catkinized version of the orb_slam_2 library:
+This package integrates orb_slam_2 into ROS in a more user friendly way than what is offered by the original library [ORB_SLAM2](https://github.com/raulmur/ORB_SLAM2)
```

Comment:
I would leave the qualifier in here, "in what I believe to be a more user friendly way".